itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -3497,6 +3498,57 @@ public void testAuthForNotificationAPIs() throws Exception {
     }
   }
 
+  @Test
```

Comment:
Also, need to check if rename temp table/partition also doesn't archive to cmpath. If not, please raise separate ticket for this.

There is another case of "Exchange partition" which need to archive moved partition files to cmpath in case of non-temp tables. Need to check this as well and shall raise separate ticket if this is a bug.

Reply:
rename temp table is already there in the test case ..

Reply:
Yes, rename partition on temp table and exchange partition need to be checked.